olive/systems/azureml/aml_pass_runner.py

```diff
@@ -102,6 +102,7 @@ def main(raw_args=None):
 
     # run pass
     output_model = p.run(input_model, None, output_model_path)
+    output_model.set_model_attributes(input_model.model_attributes)
```

Comment:
>output_model.set_model_attributes(input_model.model_attributes) [](http://example.com/codeflow?start=4&length=63)

needless? I suppose the p.run will automatically propagate the model config?

Reply:
Yes, removed.